Plugins/Flow.Launcher.Plugin.Explorer/Search/SearchManager.cs

```diff
@@ -30,12 +31,13 @@ private class PathEqualityComparator : IEqualityComparer<Result>
 
             public bool Equals(Result x, Result y)
             {
-                return x.Title == y.Title && x.SubTitle == y.SubTitle;
+                return x.Title.Equals(y.Title, StringComparison.OrdinalIgnoreCase) 
+                    && string.Equals(x.SubTitle, y.SubTitle, StringComparison.OrdinalIgnoreCase);
             }
 
             public int GetHashCode(Result obj)
             {
-                return HashCode.Combine(obj.Title.GetHashCode(), obj.SubTitle?.GetHashCode() ?? 0);
+                return HashCode.Combine(obj.Title.ToLowerInvariant(), obj.SubTitle?.ToLowerInvariant() ?? "");
```

Comment:
Is the GetHashCode not required? 

Why is the return type from the null check changed?

Reply:
it's not required. using hashcode.combine is to generating hash code based on multiple values, not to hash their hash code again.